src/main/java/Parser.java

```diff
@@ -0,0 +1,90 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Parser {
+
+    // method to make sense of command and then act on that command
+
+    /**
+     * Function that makes sense of the command.
```

Comment:
In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.). Instead of starting with "Function", you can immediately start with "Makes sense...". There should also be an empty line between description and parameter section.